event-sourcing/event-source/pom.xml

```diff
@@ -19,6 +19,7 @@
         <dependency>
             <groupId>uk.gov.justice.framework-api</groupId>
             <artifactId>framework-api-event-source</artifactId>
+            <version>${framework-api.version}</version>
```

Comment:
This shouldn't be necessary; the version is managed via the framework-api-bom

Reply:
fixed